src/main/pages/che-7/overview/proc_installing-che-on-openshift-4.adoc

```diff
@@ -8,78 +8,77 @@
 
 To install Che on OpenShift 4:
 
-== Creating the che project
+. Create the che project:
 
-. Create a project named *che* using the *Projects tab*.
-. Click the *Create Project* button.
+.. Click the *Projects* tab and click the btn:[Create Project] button.
 +
 image::installation/open-shift-create-project.png[link="{imagesdir}/installation/open-shift-create-project.png"]
 
-. In the *Name* field, type `che` and click the btn:[Create] button.
+.. In the *Name* field, type _che_ and click the btn:[Create] button.
```

Comment:
for typed input, we use monospace

Reply:
Looks like I have been living in the studio days when I was asked to use italics for user input. Now know that we use monospace. Shall keep in mind. Thanks!